snippets/paging-cursor.js

```diff
@@ -0,0 +1,48 @@
+// the perform method of our trigger
+// ensure operation.canPaginate is true!
+
+const performWihoutAsync = (z, bundle) => {
+  return Promise.resolve()
+    .then(() => {
+      if (bundle.meta.page === 0) {
+        // first page, no need to fetch a cursor
+        return Promise.resolve();
+      } else {
```

Comment:
unnecessary `else` — since the `if` has a `return`, you can just `return z.cursor.get()` directly at the top of this `.then()` callback's scope

Reply:
fair point. I know that's what we usually do, but in the interest of simplicity, i'm going to leave as-is unless you care a lot

Reply:
fine by me to leave as-is 👍